scRNA-seq/01-filtering_scRNA-seq.Rmd

```diff
@@ -0,0 +1,420 @@
+---
+title: "Single cell RNA-seq quality control and filtering"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2019**
+
+In this notebook, we'll perform quality control analyses of scRNA-seq count data. 
+The [data set](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465) we are using is cells from a glioblastoma sample that were Fluorescence-Activated Cell sorted (FACS) and then processed by paired end sequencing of RNA using the [Smart-seq2 protocol](http://www.nature.com/doifinder/10.1038/nmeth.2639) 
+[(Darmanis *et al.* 2017)](https://www.ncbi.nlm.nih.gov/pubmed/29091775).
+
+  
+## Set Up 
+
+```{r setup}
+# Magrittr for the pipe %>%
+library(magrittr)
+
+# GGPlot2 for the plots
+library(ggplot2)
+```
+
+## Import single-cell RNA-seq counts matrix
+
+The Smart-seq2 sequencing results in separate paired fastq files for each processed cell (in this case over 7000 total fastq files).
+We have processed this data with `salmon quant` and used `tximport` to create a gene-level count matrix, filtering samples to only include those with at least 30% of reads mapping.
+
+![](diagrams/full-length_1.png) 
+
+We will discuss *tag-based* scRNA-seq data (such as 10X), in a later notebook,  where you will learn how to process raw data to a count matrix and could apply these same types of steps to filter and normalize the data.
+
+### Directories and files
+
+The data files we will be using for this part of the project are in the `data/glioblastoma` subdirectory of the `scRNA-seq` directory where this file is located.
+The main files are further located in the `preprocessed` subdirectory (and sometimes deeper than that!). 
+A figure of the (partial) directory structure for this module is shown below.
+
+![](diagrams/glioblastoma_dir_structure.png)
+
+Importantly, the `preprocessed` directory is actually a *link* (think of a shortcut or alias) to a shared folder, so everyone is looking at exactly the same sets of files in there, and you will not be able to write to that directory.
+This is actually a good idea/practice to enforce in your own projects, if you can.
```

Comment:
Combining these two sentences to get to your point would be nice. 
```suggestion
It's good practice to store your data in a place that you should never write to, except when adding new data.
```

Reply:
Yeah, I like this.